Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerQuoteBubbleView.swift

```diff
@@ -61,53 +69,79 @@ open class _ChatMessageComposerQuoteBubbleView<ExtraData: ExtraDataTypes>: View,
 
         authorAvatarView.contentMode = .scaleAspectFit
         
-        attachmentPreview.layer.cornerRadius = attachmentPreviewWidth / 4
+        attachmentPreview.layer.cornerRadius = attachmentPreviewSize.width / 4
         attachmentPreview.layer.masksToBounds = true
         
-        container.centerStackView.layer.cornerRadius = 16
-        container.centerStackView.layer.borderWidth = 1
-        container.centerStackView.layer.borderColor = uiConfig.colorPalette.messageComposerBorder.cgColor
-        container.centerStackView.layer.masksToBounds = true
+        contentView.layer.cornerRadius = 16
+        contentView.layer.borderWidth = 1
+        contentView.layer.borderColor = uiConfig.colorPalette.messageComposerBorder.cgColor
+        contentView.layer.masksToBounds = true
     }
     
     override open func setUpLayout() {
-        embed(container)
-        
         preservesSuperviewLayoutMargins = true
         
-        container.preservesSuperviewLayoutMargins = true
-        container.isLayoutMarginsRelativeArrangement = true
-        
-        container.leftStackView.isHidden = false
-        container.leftStackView.addArrangedSubview(authorAvatarView)
-        authorAvatarView.widthAnchor.pin(equalToConstant: avatarViewWidth).isActive = true
-        authorAvatarView.heightAnchor.pin(equalTo: authorAvatarView.widthAnchor, multiplier: 1).isActive = true
-        
-        container.centerContainerStackView.spacing = UIStackView.spacingUseSystem
-        container.centerContainerStackView.alignment = .bottom
+        addSubview(container)
         
-        container.centerStackView.isLayoutMarginsRelativeArrangement = true
-        container.centerStackView.layoutMargins = layoutMargins
+        container.spacing = UIStackView.spacingUseSystem
+        container.alignment = .bottom
         
-        container.centerStackView.isHidden = false
-        container.centerStackView.spacing = UIStackView.spacingUseSystem
-        container.centerStackView.alignment = .top
-        container.centerStackView.addArrangedSubview(attachmentPreview)
-
-        attachmentPreview.widthAnchor.pin(equalToConstant: attachmentPreviewWidth).isActive = true
-        attachmentPreview.heightAnchor.pin(equalTo: attachmentPreview.widthAnchor, multiplier: 1).isActive = true
-
-        container.centerStackView.addArrangedSubview(textView)
+        containerConstraints = [
+            container.leadingAnchor.pin(equalTo: leadingAnchor, constant: directionalLayoutMargins.leading),
+            container.trailingAnchor.pin(equalTo: trailingAnchor, constant: -directionalLayoutMargins.trailing),
+            container.topAnchor.pin(equalTo: topAnchor, constant: directionalLayoutMargins.top),
+            container.bottomAnchor.pin(equalTo: bottomAnchor, constant: -directionalLayoutMargins.bottom)
+        ]
```

Comment:
We can pin the constraints to `layoutMarginsGuide` but in this case it will be harder to customize it with just nullifying or changing constants.